native/src/utils/renderHtml.ts

```diff
@@ -5,237 +5,259 @@ import { ERROR_MESSAGE_TYPE, getFontFaceSource, HEIGHT_MESSAGE_TYPE, WARNING_MES
 
 // language=JavaScript
 const renderJS = (cacheDictionary: Record<string, string>) => `
-  function reportError (message, type) {
-    if (!window.ReactNativeWebView) {
-      return window.setTimeout(function() { reportError(message, type) }, 100)
-    }
+    function reportError (message, type) {
```

Comment:
Please revert the formatting, now there are 4 spaces instead of our default of 2. Imo the previous was correct. I think this new indentation is just coming from the fact that you added brackets around the string/didn't use direct return.

Reply:
@steffenkleinle can you specify which string you mean? :)